### ISSUE TITLE
Regions named through the use of landmarks or semantic HTML (header, main, nav and footer)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/purl#753]
Named regions can be accomplished through [ARIA roles/labels](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11.html) or through semantic HTML elements, such as header, main and footer. PURL pages have a footer element, but they're missing header and main elements. I suggest adding header and main elements. It was flagged by SiteImprove under ARIA, but it's also required to meet compliance for [WCAG 2.1 A Understanding Docs](https://www.w3.org/WAI/WCAG21/Understanding/) requirements - [Info and Relationship](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) and [Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks).

Alan also mentions it in [Accessibility Process - Developers (WIP/DRAFT)](https://docs.google.com/document/d/14vsLwKB4BiZZnkcsrj8r9hAa9w1hHS6IvD6sKDr7jPo/edit).

**Comment by justinlittman** [comment]
@dbranchini Can you identify what would be the header and main sections?

**Comment by dbranchini** [comment]
The divs with the following id's should be included in the header element:
#topnav-container
#feedback-form
#search-navbar-container

I see there's already a section with the role of "main," which satisfies the requirement for a main element.

**Comment by justinlittman** [comment]
Looks like there is already a header tag. Is this correct?

![image](https://github.com/sul-dlss/purl/assets/588335/5b609f78-bc0b-40f5-b69f-d894cbdef65b)

**Comment by dbranchini** [comment]
Yes, I see that now. It's likely that this issue was triggered because the header, footer, and main page sections are not directly under the body tag. Is that too much of a refactor? It's probably not a high priority if it is.

**Comment by justinlittman** [comment]
Looking at the code I would be really hesitant to make those changes, especially since I'm completely unfamiliar with the styling.

**Comment by justinlittman** [comment]
Closing. If you disagree, feel free to re-open.

**Comment by justinlittman** [comment]
Jinx ... I just re-found this and proposed a fix: #848

**Comment by lwrubel** [comment]
Ha! Thanks

**Comment by dbranchini** [comment]
I apologize for any lack of clarity, but above I mentioned these three improvements. I looks like the last one was addressed. Thank you! But the extra "contentinfo" on the footer is still there. Once you remove that, we should be all set since the remaining two flag refer to the same thing.

- [ ] Document should not have more than one contentinfo landmark (For some reason, in the footer landmark, there's also a role=contentinfo landmark. Role essentially serves the same purpose as the HTML element footer and they're conflicting.)

- [ ] Ensure landmarks are unique. Again this refers to the contentinfo role on the footer. There's also two footers on this page - the Stanford Libraries footer and the Stanford University footer. In this article - https://www.w3schools.com/accessibility/accessibility_landmarks.php - thee guidance is that you can have two elements, but then they must be described more fully with an aria-label. For example, and .

- [X] All page content should be contained by landmarks. This is referring to the or the red banner displaying the text "Stanford Digital Repository." I believe it should be contained in the landmark, although the id name suggests it might be a navigation element.

**Comment by justinlittman** [comment]
@dbranchini I'm unclear how to fix this problem. Here's snippets of the existing relevant HTML:
```
<footer role="contentinfo">
  <div id="sul-footer-container">
    <! -- SUL footer -->
  </div>
</footer>
<div id="global-footer" role="contentinfo">
  <div class="container">
    <! -- Global footer -->
  </div> <!-- .container end -->
</div> <!-- global-footer end -->
```

**Comment by justinlittman** [comment]
Oh, I can do it like H2:
<img width="371" alt="image" src="https://github.com/sul-dlss/purl/assets/588335/02154b00-1171-4bb5-8af1-a8fa38e256db">

**Comment by dbranchini** [comment]
Looks good, @justinlittman, I'm closing. Thank you!

**Comment by justinlittman** [comment]
PR needs to get merged first. Submitting for review now.
